spec/rubocop/cop/style/min_max_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Style::MinMax, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'with an array literal containing calls to `#min` and `#max`' do
+    context 'when the expression stands alone' do
+      it 'registers an offense if the receivers match' do
+        expect_offense(<<-RUBY.strip_indent)
+          [foo.min, foo.max]
+          ^^^^^^^^^^^^^^^^^^ Use `foo.minmax` instead of `[foo.min, foo.max]`.
+        RUBY
+      end
+
+      it 'does not register an offense if the receivers do not match' do
+        expect_no_offenses(<<-RUBY.strip_indent)
+          [foo.min, bar.max]
+        RUBY
+      end
+
+      it 'does not register an offense if there are additional receivers' do
```

Comment:
In this test there is only one receiver, `foo`. Perhaps reword as “does not register an offense if there are additional elements in the array”?

Reply:
Good catch!